lib/senkyoshi/models/file_resource.rb

```diff
@@ -19,5 +19,19 @@ def self.from(xml, pre_data, _resource_xids = nil)
     def iterate_xml(_xml, _pre_data)
       self
     end
+
+    def create_module(course)
+      course.canvas_modules ||= []
+      cc_module = Course.master_module(course)
```

Comment:
Yeah, I tried to do it a different way, but I just couldn't think of a better way to make it all flow better. Too many files and too many tools to figure out how to deal with them cleanly. But I am totally open to ideas.